packages/venia-concept/src/components/Header/header.js

```diff
@@ -53,7 +60,27 @@ const Header = props => {
     ) : null;
 
     const { formatMessage } = useIntl();
-    const title = formatMessage({ id: 'logo.title', defaultMessage: 'Venia' });
+    const title = formatMessage({ id: 'logo.title', defaultMessage: 'Drexel' });
+
+    //The Intersection Observe of a header
+    useEffect(() => {
+        const observer = new IntersectionObserver(
+            (entries) => {
+                entries.forEach((entry) => {
+                    if(headerRef.current && !entry.isIntersecting) {
+                        headerRef.current.classList.add('header-animation', 'top-0', 'sticky');
```

Comment:
Can we make it works a little different ?
This add action is one-directional.
We want to add and remove classes based on header position.
In this case, those styles will remain permanent. We don't want to have the header transparent at the very top position.
As an example, You can check how '.container-header-sticky' changing here: https://blueskytechmage.com/drexel/en2/.

Reply:
PS: please, use only module classes, like: classes['header-animation'], classes['top-0'], classes. sticky.